pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -14,6 +14,10 @@ from cpython.object cimport (
     PyObject,
     PyObject_RichCompare,
 )
+from libc.limits cimport (
+    INT_MAX,
```

Comment:
elsewhere we use e.g. util.INT64_MAX.  is there a reason for using INT_MAX instead of INT64_MAX?

Reply:
The CPython implementation itself just stores microseconds as an `int`

https://github.com/python/cpython/blob/7fc51c3f6b7b13f88480557ff14bdb1c049f9a37/Include/datetime.h#L40